hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/managed/ManagedObject.java

```diff
@@ -27,8 +29,17 @@
 class ManagedObject<T extends RocksObject> implements AutoCloseable {
   private final T original;
 
+  private final StackTraceElement[] elements;
+
+  public static final Logger LOG = LoggerFactory.getLogger(ManagedObject.class);
+
   ManagedObject(T original) {
     this.original = original;
+    if (LOG.isDebugEnabled()) {
```

Comment:
nit: This separates the decision of creating and logging stacktrace to 2 different authorities/configs.  I guess ManagedRocksObjectUtils can export some method like `isStacktraceDebugEnabled` and it can be the only authority of determining if stacktrace should be created and logged.

Reply:
Done.